### ISSUE TITLE
.editorconfig when_on_single_line gives an error saying it's an invalid value

### ISSUE THREAD

**Issue by eyalsk** [dotnet/roslyn#24251]
**Version Used**: Visual Studio Community v15.5.3

**Steps to Reproduce**:

Add the following rules:
```
csharp_style_expression_bodied_operators = when_on_single_line:error
csharp_style_expression_bodied_indexers = when_on_single_line:error
csharp_style_expression_bodied_accessors = when_on_single_line:error
csharp_style_expression_bodied_methods = when_on_single_line:error
csharp_style_expression_bodied_properties = when_on_single_line:error
```

**Expected Behavior**:

1. Should not cause an error.

2. Should show `when_on_single_line` in the intellisense.

**Actual Behavior**:

1. `when_on_single_line` causes the following error "EC114: **whenon**_single_line is not a valid value for..."; notice the typo.

2. Doesn't show the value in the intellisense.

**Comment by sharwell** [comment]
> Should show `when_on_single_line` in the intellisense.

There is no real code completion functionality for **.editorconfig** right now.

> `when_on_single_line` causes the following error "EC114: whenon_single_line is not a valid value for..."; notice the typo.

This is reported by an extension for Visual Studio: https://github.com/madskristensen/EditorConfigLanguage

**Comment by eyalsk** [comment]
@sharwell Oh, thank you.
